homeassistant/components/wunderlist/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""
+Component to interact with Wunderlist.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/wunderlist/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_NAME, CONF_ACCESS_TOKEN)
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['wunderpy2==0.1.6']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'wunderlist'
+CONF_CLIENT_ID = 'client_id'
+CONF_LIST_NAME = 'list_name'
+CONF_STARRED = 'starred'
+
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_CLIENT_ID): cv.string,
+        vol.Required(CONF_ACCESS_TOKEN): cv.string
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+SERVICE_CREATE_TASK = 'create_task'
+
+SERVICE_SCHEMA_CREATE_TASK = vol.Schema({
+    vol.Required(CONF_LIST_NAME): cv.string,
+    vol.Required(CONF_NAME): cv.string,
+    vol.Optional(CONF_STARRED): cv.boolean
+})
+
+
+def setup(hass, config):
+    """Set up the Wunderlist component."""
+    conf = config[DOMAIN]
+    client_id = conf.get(CONF_CLIENT_ID)
+    access_token = conf.get(CONF_ACCESS_TOKEN)
+    entity = Wunderlist(access_token, client_id)
+    if not entity.check_credentials():
+        _LOGGER.error("Invalid credentials:\n client_id=%s\n "
+                      "access_token=%s", client_id, access_token)
```

Comment:
`_LOGGER.error("Invalid credentials")` is enough. Don't print usernames, passwords, API keys or token to log file.